packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/EmailAddress/index.js

```diff
@@ -47,8 +50,8 @@ class EmailAddressContainer extends React.PureComponent {
   }
 
   handleChangeEmailView () {
+    const { email } = this.props.data.getOrElse({})
     this.props.updateUI({ changeEmailToggled: true })
-    const email = this.props.data.data.email
     this.props.formActions.change('securityEmailAddress', 'changeEmail', email)
```

Comment:
Should you still do that if email is undefined?

Reply:
When is email undefined?

Reply:
Whenever this.props.data is not a Remote.Success

Reply:
I guess that happens if we fail to fetch the settings or if they haven't been fetched yet

Reply:
I think settings are fetched when user logs in.

If they reach security center and settings are not fetched the worst I think will happen is the field will be empty, instead of prefilled with your email by default.

Anyway, think this is better than data.data, which would break HMR